.github/workflows/packaging.yml

```diff
@@ -40,6 +40,14 @@ jobs:
       DIST: ${{ matrix.platform.dist }}
 
     steps:
+      - name: Prepare checkout
```

Comment:
It's always good to self-review your PR, at least briefly. here we definitely have a traces of the old approach.

Reply:
Deleted this step, now `Version check` after `Clone the module`